### PR TITLE
test: add text knobs for bar

### PR DESCRIPTION
**Pull request by stefanoScalzo**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/1954
#### Please provide a brief summary of this pull request.
This or will add text knobs for left,middle and right section of the bar component.
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by InnaAtanasova** [comment]
What's the value of this change? This is just text. We are not testing any functionality with this.

**Comment by stefanoScalzo** [comment]
> What's the value of this change? This is just text. We are not testing any functionality with this.

Anything that we should be able to change such as text or anything else related to that component needs to have a knob

**Comment by stefanoScalzo** [comment]
> I don't see any value

It could be testing te width of the text, ellipses whatever else. 
In the breadcrumb example, I had to make sure all the texts had knobs so I am doing the same thing here

**Comment by stefanoScalzo** [comment]
> I see value of it, as Stefano said it allows to test truncation/overflow of text. I would keep these changes

@droshev 
Screen Shot 2020-04-28 at 9 40 57 AM

The text is inside a bar-element and belongs to the bar component. This will test the full width spec of the fd-bar-element

**Comment by salarenko** [comment]
I agree with @JKMarkowski and @stefanoScalzo. If there is a certain way how text should behave in Bar, then we should have a possibility to test it.

File: src/stories/bar/fd-bar.stories.ts [hunk 1]
```diff
@@ -91,6 +91,15 @@ export const Bar = () => ({
...
+<<<<<<< HEAD
         fullwidthVar: boolean('Full width element', true)
+=======
+        fullwidthVar: boolean('Full width element', true),
+        leftText: text('Left Section Text', 'Left Section with very very very very very very very very very very very very very very very very very very  very very very very very very long text'),
+        middleText: text('Middle Section Text', 'Section'),
+        rightText: text('RightSection Text', 'Section'),
+
+        
+>>>>>>> test:add text knobs for bar
```
Comment (InnaAtanasova) [inline src/stories/bar/fd-bar.stories.ts:103]: Leftovers from rebase

**Pull request closed without merging**